README.md

```diff
@@ -123,15 +123,15 @@ the most recent citation.
 
 ## License and acknowledgments
 
-[PyMAPDL](https://mapdl.docs.pyansys.com/version/stable/) is licensed under
+[PyMAPDL](https://mapdl.docs.pyansys.com/) is licensed under
```

Comment:
Thanks for updating these links. The more references to the base URL we have, the better.